packages/frontend/src/pages/drop-and-fusion.game.vue

```diff
@@ -1095,13 +1095,21 @@ function attachGameEvents() {
 		dropReady.value = false;
 		isGameOver.value = true;
 
-		misskeyApi('bubble-game/register', {
-			seed,
-			score: score.value,
-			gameMode: props.gameMode,
-			gameVersion: game.GAME_VERSION,
-			logs: DropAndFusionGame.serializeLogs(logs),
-		});
+		if (
+			os.confirm({
```

Comment:
```suggestion
			await os.confirm({
```
こんな感じだと思う(未検証なのでapply suggestion非推奨)

Reply:
ローカル環境で試しましたがそれでもlint失敗してしまいますね...
さっきとは別のエラー、`Expected non-Promise value in a boolean conditional  @typescript-eslint/no-misused-promises`です。

Reply:
os.confirmの結果を変数に収納することで治りました。